x-pack/plugins/security_solution/public/flyout/document_details/right/components/correlations_overview.tsx

```diff
@@ -104,16 +127,8 @@ export const CorrelationsOverview: React.FC = () => {
             defaultMessage="Correlations"
           />
         ),
-        link: {
-          callback: goToCorrelationsTab,
-          tooltip: (
-            <FormattedMessage
-              id="xpack.securitySolution.flyout.right.insights.correlations.overviewTooltip"
-              defaultMessage="Show all correlations"
-            />
-          ),
-        },
-        iconType: 'arrowStart',
+        link,
+        iconType: !isPreviewMode ? 'arrowStart' : undefined,
```

Comment:
I'm not convinced we should remove the icon here, I know there is a discussion on Slack. Let's see what UIUX says

Reply:
after discussing with Paul (and UI seems aligned), let's remove it!